### PR TITLE
[trello-92]: extrapolation update

### PR DESCRIPTION
**Pull request by Janbinkowski96**

### CODE REVIEW

File: app/apmc/ds/processing/data_processing.py [hunk 1]
```diff
@@ -44,10 +44,12 @@ def data_set_split(X_array, y_vector, normalization=False):
 
         mean_array = scaler.mean_  # data used for scaling user input
         std_array = scaler.scale_
+
```
Comment (szymon6927) [inline app/apmc/ds/processing/data_processing.py]: no new line here

File: app/apmc/ds/processing/data_processing.py [hunk 2]
```diff
@@ -44,10 +44,12 @@ def data_set_split(X_array, y_vector, normalization=False):
 
         mean_array = scaler.mean_  # data used for scaling user input
         std_array = scaler.scale_
+    
     else:
         X = X_array
-        mean_array = None
-        std_array = None
+
```
Comment (szymon6927) [inline app/apmc/ds/processing/data_processing.py]: is this newline also necessary?

File: app/apmc/ds/processing/data_processing.py (hunk 1 above)
Comment (Janbinkowski96) [reply app/apmc/ds/processing/data_processing.py]: ok

File: app/apmc/ds/processing/data_processing.py (hunk 2 above)
Comment (Janbinkowski96) [reply app/apmc/ds/processing/data_processing.py]: nope

**Pull request merged**